bdb/tran.c

```diff
@@ -1567,6 +1568,7 @@ static int bdb_tran_commit_with_seqnum_int_int(
                 outrc = -1;
                 atexit(abort_at_exit);
                 failed_to_log = 1;
+                bdb_osql_trn_repo_unlock();
```

Comment:
This is definitely right, though looks like a different problem.  What problem was this?

Reply:
Didn't occur during testing- I just noticed it was broken.